### ISSUE TITLE
Health Code Freezes When Mount A Horse!

### ISSUE THREAD

**Issue by duy64** [joffnerd/botw-trainer#26]
Infinite health code freezes when I mount a horse.
This is a USA version of the game.

**Comment by duy64** [comment]
Using v2.3.2 of the trainer

**Comment by joffnerd** [comment]
I haven't tried this yet but as I mention a lot i am Pal/EUR so I cant test for USA i'm afraid.

**Comment by voidwards** [comment]
I'm having the same issue with 2.3.3 with US version. The entire console will lock up. I did notice that when looking at the values for health in the trainer it changes depending on whether you're on a horse or not, which may be related. For example off the mount it read 76, then when mounted I got something like 432. 

The error I received the last time it froze was:

Exception of type 'BotwTrainer.ETCPGeckoException' was thrown.   at BotwTrainer.TCPGecko.Dump(UInt32 startdump, UInt32 enddump, Stream[] saveStream)
   at BotwTrainer.TCPGecko.peek(UInt32 address)
   at BotwTrainer.MainWindow.DebugData()
   at BotwTrainer.MainWindow.SaveClick(Object sender, RoutedEventArgs e)
